README.md

```diff
@@ -1,59 +1,213 @@
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ](#faq)
+- [📝 License](#license)
 
-[Live Demo Link](https://ameerhamzakhalid.github.io/portfolio-ameer-hamza/)
+<!-- PROJECT DESCRIPTION -->
 
-## Getting Started
+# 📖 [your_project_name] <a name="about-project"></a>
+A simple site to keep a record of awesome books.
 
+## 🛠 Built With <a name="built-with"></a>
+- HTML
+- CSS
+- JavaScript
+
+### Tech Stack <a name="tech-stack"></a>
+
+> This project utilizes the following stack
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://html.com">HTML</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li><a href="https://javascript.com">JavaScript</a></li>
+  </ul>
+</details>
+
+<details>
+<summary>Frontend</summary>
+  <ul>
+    <li><a href="https://www.postgresql.org/">CSS</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **Add a book**
+- **persistent Books record**
+- **Remove a book from the list**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Coming Soon](#)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+```sh
+git clone [this repo]
+```
 
-For copying and running follow the steps below.
-- Clone the repository by excecuting the git clone ```https://github.com/ameerhamzakhalid/awesome-books``` from the terminal.
 ### Prerequisites
-  - Git installed
 
+In order to run this project you need:
+
+```sh
+open index.html
+```
+### Deployment
 
-  - open index.html
+You can deploy this project using:
 
-## Authors
+> Coming soon
 
-👤 **Ameer Hamza**
+<!--
+Example:
 
-- GitHub: [@ameerhamzakhalid](https://github.com/ameerhamzakhalid)
-- Twitter: [@twitterhandle](https://twitter.com/ameeerhamza1997)
-- LinkedIn: [Ameer Hamza](https://www.linkedin.com/in/choudhary-hamza-37b17a141/)
+```sh
+
+```
+ -->
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
 
 👤 **Jared Atandi**
 
-- GitHub: [@github](https://github.com/jaredatandi)
-- Twitter: [@twitter](https://twitter.com/@one_good_man_)
+- GitHub: [@githubhandle](https://github.com/jaredatandi)
+- Twitter: [@twitterhandle](https://twitter.com/one_good_man)
 - LinkedIn: [LinkedIn](https://linkedin.com/in/engineerjaredatandi)
 
-## 🤝 Contributing
+👤 **Hamza**
+
+- GitHub: [@githubhandle](https://github.com/)
+- Twitter: [@twitterhandle](https://twitter.com/)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/)
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
```

Comment:
Currently, your readme doesn’t contain all of your personal information. Please take into account that the readme is the showcase of your project 😉. In this file, you are going to describe what you built and any additional information. You should use the [Microverse Template](https://github.com/microverseinc/readme-template), but don't forget to personalize it with your data 😄! Add the project title, a description that you like, your social media links, and anything else that you want to share.